graphql/ui_plugin_build_baron.go

```diff
@@ -48,6 +48,10 @@ func BbFileTicket(context context.Context, taskId string, execution int) (bool,
 		taskNotFound = true
 		return taskNotFound, errors.Wrap(err, fmt.Sprintf("task not found for id %s", taskId))
 	}
+	if t.Details.Description == evergreen.TaskDescriptionStranded {
```

Comment:
I _think_ we don't want to do this here; we call this functions when users specifically choose to file a ticket, in which case we shouldn't do any special casing (cc @malikchaya2 to check that)

Reply:
That's correct, this is the function that gets called when a user clicks on the "File Ticket" button on a task. You can see the button here on [legacy](https://evergreen.mongodb.com/task/evergreen_windows_test_agent_util_fcfa25ab6daddff116ad7260d9bd508aa7e83055_21_07_09_20_03_19) and here and [spruce](https://spruce.mongodb.com/task/evergreen_windows_test_agent_util_fcfa25ab6daddff116ad7260d9bd508aa7e83055_21_07_09_20_03_19/annotations?execution=0). I agree that we shouldn't do any special casing here, because the user will expect to see the ticket filed no matter the state.

Reply:
It looks like this comment is till open

Reply:
@hadjri if you're waiting on a new review I think we're waiting for this to be addressed

Reply:
My mistake, had thought the review was still pending. Re-requested.